### ISSUE TITLE
Awair shows two devices

### ISSUE THREAD

**Issue by matrover** [biemond/com.athom.awair#1]
Hi Biemond,

Thanks a ton for this app! really was waiting for this one. 

When i want to add the Awair device it shows 2 devices (i only have one). 
im in Homey FW: 1.5.13
app: V0.0.1

**Comment by biemond** [comment]
Hi, 

oh really, interesting. I am calling the device api.  I will send you a curl statement which you can try yourself with your bearer token. or share me your bearer token.  biemond at gmail 

also now I am polling every 2 min but I got an error I was polling too much in the last 25 hours. Fixing that now.

**Comment by matrover** [comment]
ok, i've send you the token. 

Thanks for looking in to the polling as well i didnt get any updates on vaules as well.

**Comment by biemond** [comment]
your output 
```
{
    "devices": [
        {
            "name": "Awair",
            "latitude": .,...
            "preference": "GENERAL",
            "timezone": "Europe/Amsterdam",
            "roomType": "LIVING_ROOM",
            "deviceType": "awair-r2",
            "longitude": .5172,..
            "spaceType": "HOME",
            "deviceUUID": "awair-r2....",
            "deviceId": 1051,
            "locationName": "Go..."
        }
    ]
}
```
I see 1 device called awair

**Comment by biemond** [comment]
also I am running homey v2

**Comment by biemond** [comment]
He, can try the latest 0.0.3. else it can a homey v2 issue.

**Comment by matrover** [comment]
Still 2 devices: 
![image](https://user-images.githubusercontent.com/18408057/52962147-d0a68700-339c-11e9-8d03-0b8c023b70b2.png)

Additional: 
i cant see the values on the device: 
![image](https://user-images.githubusercontent.com/18408057/52962369-5fb39f00-339d-11e9-8dc2-a71bd7b73cbe.png)

But i can see them in the insights: 
![image](https://user-images.githubusercontent.com/18408057/52962419-7fe35e00-339d-11e9-92bc-8d0c0c8adecf.png)

but maybe this is an v1.x issue

**Comment by biemond** [comment]
true, let me check again.

Can you do a homey upgrade or will that be a big issue for you.

**Comment by biemond** [comment]
also there is now a homey bug in v2 if you have 2 devices and using crontasks (1 should be fine) It will ignore the oldest device , only the last is updated.

**Comment by matrover** [comment]
If its working fine on 2.0, than dont bother too much. Im waiting for athom to fix the BLE bug on 2.0 and then i will update anyway

**Comment by matrover** [comment]
Yep, no issue with 2.0. only one device shows up and works. thnx
